docs/UserGuide.md

```diff
@@ -185,13 +207,13 @@ Only products added already via `addProduct` can be used! This is to help ensure
 
 And there we have it! We have just gone through the basic bookkeeping features of *MyInsuRec*. Beyond that, we also have more advanced features such as getting a quick glance of all your upcoming meetings, filtering clients by products. To find out and learn more about these features, head to the [Features section](#5-features) where all the commands and their details can be found.
 
-To get started with an clean state of *MyInsuRec*, type in the command `clear`. This removes all the sample data from *MyInsuRec*, so don't get panicked! 
+To get started with an clean state of *MyInsuRec*, type in the command `clear`. This removes all the sample data from *MyInsuRec*, so don't get panicked!
```

Comment:
don't panick might be a better description?

Reply:
`a clean state` instead of `an clean state`

Reply:
will make this change!